src/sentry/api/urls.py

```diff
@@ -1502,6 +1503,11 @@ def create_group_urls(name_prefix: str) -> list[URLPattern | URLResolver]:
         OrganizationMetricsExtractionRulesEndpoint.as_view(),
         name="sentry-api-0-organization-metrics-extraction-rules",
     ),
+    re_path(
+        r"^(?P<organization_id_or_slug>[^\/]+)/(?P<project_id_or_slug>[^\/]+)/metrics-usage/(?P<span_attribute>[^\/]+)/$",
+        ProjectMetricsUsageEndpoint.as_view(),
+        name="sentry-api-0-project-metrics-usage",
+    ),
```

Comment:
This should probably be moved to the PROJECT_URLS instead of ORGANIZATION_URLS: https://github.com/getsentry/sentry/blob/7756b739c0bdedc8840fcc799999a5da4e54999f/src/sentry/api/urls.py#L2180

Reply:
True, good catch